### DISCUSSION TITLE
API Search

### DISCUSSION THREAD

**Discussion by SHELA** [postalserver/postal#2296] [Feature suggestions]
Hello,
As a feature idea to have API search in income and outgoing messages.
The same that in UI but from API

**Comment by willpower232** [comment]
I don't know what the new API will be capable of but Postal is not intended to store messages long term so if you're trying to search an archive of messages, I would say that it would be better to implement that in whatever you're using to send emails via Postal.

**Comment by Igcorreia** [comment]
@willpower232 I need a way to find an email by "to:". Example:

myname+something@domain.com

I would like to use the API to search for email: "to:something@domain.com"

Is this possible?
